app/linkedList.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by tatsiak on 4/22/17.
+ */
+'use strict';
+
+class Node {
+    constructor(value, previous = null) {
+        this._value = value;
+        this._previous = previous;
+        this._next = null;
+    }
+}
+class LinkedList {
+    constructor() {
+        this._start = null;
+        this._end = null;
```

Comment:
It's possible, but it'll make logic more complicated.
Setting both start and end required because every non-first element should have _next and _previous.
So if I decide to left only start - I'll have to change logic of all methods without any grow in performance.
That's why I think I should keep it as it is now.